From 6196cf77f62a5cdaedc473fe46702e8705366410 Mon Sep 17 00:00:00 2001
From: YumeMichi <do4suki@gmail.com>
Date: Sun, 19 Feb 2017 21:33:53 +0000
Subject: [PATCH] Fix build on LineageOS

---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 4cbd5bc..70fc513 100644
--- a/Android.mk
+++ b/Android.mk
@@ -22,7 +22,7 @@ LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
 LOCAL_PACKAGE_NAME := AmazeFileManager
-LOCAL_CERTIFICATE := mokee
+LOCAL_CERTIFICATE := platform
 LOCAL_MODULE_TAGS := optional
 LOCAL_PROGUARD_FLAG_FILES := proguard.flags
 
