From 64c3542fae1d1ad0a6eaa4f8535be8f863ed4d0f Mon Sep 17 00:00:00 2001
From: Jake Weinstein <jake@aospa.co>
Date: Wed, 31 Jul 2019 10:53:24 -0500
Subject: [PATCH] Define AArch32 soong variable

Change-Id: I1a52a8e8e436eec827022413270d8b7c76808a69
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 android/arch.go | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/android/arch.go b/android/arch.go
index 6e8c4bdb..cb24ff19 100644
--- a/android/arch.go
+++ b/android/arch.go
@@ -189,6 +189,7 @@ var archVariants = map[ArchType][]string{
 var archFeatures = map[ArchType][]string{
 	Arm: {
 		"neon",
+		"aarch32",
 	},
 	Mips: {
 		"dspr2",
@@ -231,9 +232,11 @@ var archFeatureMap = map[ArchType]map[string][]string{
 		},
 		"armv8-a": {
 			"neon",
+			"aarch32",
 		},
 		"armv8-2a": {
 			"neon",
+			"aarch32",
 		},
 	},
 	Mips: {
-- 
2.26.2

